### PR TITLE
use global dictionary.

### PR DESCRIPTION
**Pull request by qiwzhang**
word header file is auto generated from yaml file.
Also set global_word_count field

### CODE REVIEW

File: BUILD [hunk 1]
```diff
@@ -16,6 +16,27 @@ licenses(["notice"])
...
+    name = "global_dictionary_header_gen",
+    srcs = [
+        "@mixerapi_git//:mixer/v1/global_dictionary.yaml",
+    ],
+    outs = [
+        "global_dictionary.h",
+    ],
+    cmd = "$(location //:create_global_dictionary.py) $(location @mixerapi_git//:mixer/v1/global_dictionary.yaml) > $@",
+    tools = [
+        "//:create_global_dictionary.py",
```
Comment (qiwzhang) [inline BUILD]: Done

File: BUILD [hunk 2]
```diff
@@ -16,6 +16,27 @@ licenses(["notice"])
...
+    ],
+)
+
+cc_library(
```
Comment (qiwzhang) [inline BUILD]: Done

File: create_global_dictionary.py [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+ * limitations under the License.
+ */
+
+#ifndef MIXERCLIENT_GLOBAL_DICTIONARY_H
+#define MIXERCLIENT_GLOBAL_DICTIONARY_H
+
+namespace istio {
+namespace mixer_client {
+
+#define KGlobalDictionary { \
```
Comment (lizan) [inline create_global_dictionary.py]: Why just define but not a static std::vector?

File: create_global_dictionary.py [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef MIXERCLIENT_GLOBAL_DICTIONARY_H
+#define MIXERCLIENT_GLOBAL_DICTIONARY_H
+
+namespace istio {
```
Comment (lizan) [inline create_global_dictionary.py]: if you are just defining, you don't need namespaces.

File: create_global_dictionary.py (hunk 3 above)
Comment (qiwzhang) [inline create_global_dictionary.py]: Done,  changed to use vector.

File: create_global_dictionary.py [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+#ifndef MIXERCLIENT_GLOBAL_DICTIONARY_H
+#define MIXERCLIENT_GLOBAL_DICTIONARY_H
+
+#include <string>
+#include <vector>
+
+namespace istio {
+namespace mixer_client {
+
+static const std::vector<std::string> kGlobalWords = {
```
Comment (lizan) [inline create_global_dictionary.py]: no need `=`.

Also, make this generated file as `.cc` and also a static header.

Comment (qiwzhang) [inline create_global_dictionary.py]: Done

**Pull request merged**